examples/jest-vtu-example/nuxt.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
@paulgv you can change this to `export default` to match Nuxt 2's ESM style.

Reply:
done, thank you @galvez